releases/data-center/syskit-point-2024-5/point-2024-5-64-release-note.md

```diff
@@ -0,0 +1,87 @@
+---
+description: This article lists improvements and bug fixes in Syskit Point version 2024.5.64
+---
+
+# Syskit Point 2024.5.64
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2024.5.64 
+
+**Build number:** XX
+
+**Release date:** October 07, 2024
+
+## Get the Latest Version
+
+* **New to Syskit Point?** Get the latest Syskit Point version from [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/syskitltd.syskit\_point).\
+ Find all steps described in detail in the [deployment article](../../../set-up-point-data-center/deployment/deploy-syskit-point.md).
+* **Already using Syskit Point?** Upgrade to the latest Syskit Point version.\
+ [![](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fsyskitassetsstorage.blob.core.windows.net%2Fpoint%2FARMTemplates%2FPointUpdateDeploy%2FPointUpdateTemplate.json)
+
+See detailed step-by-step instructions on how to upgrade Syskit Point in the [upgrade article](../../../set-up-point-data-center/deployment/upgrade-syskit-point.md).
+
+## New Features
+
+* **New feature added to Provisioning!**
+  * New section available when creating or editing Templates: **Content & Structure**
+  * The Content & Structure section provides three options:
+    * **Start from scratch** - the new provisioned workspace is blank. 
```

Comment:
umjesto is blank - has the default Microsoft look and feel without any customizations.